style.css

```
@@ -0,0 +1,102 @@
+body {
+  margin: 0;
+  padding-top: 70px;
+  background: rgb(182, 223, 224);
+}
+
+button {
+  border: none;
+  background: none;
+  font-size: 17px;
+}
+
+button:hover {
+  cursor: pointer;
+}
+li {
+  padding: 5px;
+  padding-left: 20px;
+  list-style: none;
+  font-weight: light;
+  color: rgb(75, 73, 73);
+  font-size: 13px;
+}
+
+.container {
+  margin: 0 auto;
+  width: 400px;
+  height: 650px;
+  background: rgba(189, 187, 187, 0.8);
+  border-radius: 30px;
+  padding: 25px;
+}
+
+.clock {
+  margin: 0;
+  text-align: center;
+  font-weight: lighter;
+  font-size: 27px;
+}
+
+.title {
+  padding-top: 10px;
+  margin: 0;
+  font-weight: normal;
+}
+
+form {
+  text-align: center;
+}
+.toDoInput {
+  padding-left: 20px;
+  border: none;
+  width: 320px;
+  height: 40px;
+  border-radius: 15px;
+  background-color: rgb(241, 241, 229);
+}
+
+.list {
+  padding: 20px;
+  font-weight: light;
+}
+
+.toDoList,
+.doneList {
+  padding: 0;
+  padding-top: 20px;
+  margin: 0;
+  width: 350px;
+  height: 200px;
+  border-radius: 20px;
+  overflow: scroll;
```

말씀하셨던 스크롤바 영역에 생기는 흰 박스는 `overflow: scroll`로 설정하여 발생하는 문제인 듯합니다. `overflow: scroll`은 가로축, 세로축 스크롤이 모두 생기기 때문에 지금처럼 세로축 스크롤만 필요한 경우에는 `overflow-y: scroll` 또는 `overflow: auto`를 사용하시면 됩니다. 저는 개인적으로 `overflow: auto`를 선호한답니다!

```suggestion
  overflow: auto;
```

---

드디어 ...!! 해결방법을 찾았네요 감사합니다 !!!! 완전 거슬리는 부분이었는데 프짱님 덕분에 해결했어요 !! 🥰